prolude.cabal

```
@@ -35,10 +36,15 @@ library
     build-depends: 
         base >= 4.13.0 && < 4.15
         , aeson >= 1.4.7 && < 1.5
+        , array >= 0.3 && < 0.6
         , bytestring >= 0.10.10 && < 0.11
+        , containers >= 0.6.2.1 && < 0.6.3.1
```

I think this is the only dependency we need to add right now. And I would suggest a more lenient constraint. 
```suggestion
        , containers >= 0.6.2 && < 0.7
```